project/Common.scala

```diff
@@ -16,8 +16,8 @@ object Common {
   lazy val defaultSettings = buildSettings ++ sonatypeSettings
 
   lazy val buildSettings: Seq[Def.Setting[_]] = Seq(
-    scalaVersion := "2.11.12",
-    crossScalaVersions := Seq("2.11.12", "2.12.10"),
+    scalaVersion := "2.12.10",
```

Comment:
want to try `2.12.13` as latest 2.12 series?  I'd guess it is a no-op change

Reply:
Let me try that.